streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentTestUtils.java

```diff
@@ -617,21 +652,23 @@ static Map<UUID, Map<String, Optional<String>>> getRandomProcessRacks(final int
         }
         Collections.shuffle(racks);
```

Comment:
Just realizing that we might want to hand in a `Random` object as second parameter to allow us to reproduce a test run? Can you check the code for other places where we use `shuffle` to allow us to improve it across the board?

Reply:
Looks `shuffle` can take random. I think we can use one static random in this class